src/lib/Components/Inbox/Conversations/SendConversationMessage.ts

```diff
@@ -1,21 +1,30 @@
 import { commitMutation, Environment, graphql, MutationConfig, RecordSourceSelectorProxy } from "react-relay"
 import { ConnectionHandler } from "relay-runtime"
-import RelayProps from "./relayInterfaces"
+
+interface Conversation {
+  last_message_id: string
+  id: string
+  __id: string
+  from: {
+    email: string
+  }
+}
 
 export function sendConversationMessage(
   environment: Environment,
-  conversation: RelayProps["me"]["conversation"],
+  conversation: Conversation,
   text: string,
   onCompleted: MutationConfig["onCompleted"],
   onError: MutationConfig["onError"]
 ) {
-  const lastMessage = conversation.messages.edges[conversation.messages.edges.length - 1].node
-
   const storeUpdater = (store: RecordSourceSelectorProxy) => {
     const mutationPayload = store.getRootField("sendConversationMessage")
     const newMessageEdge = mutationPayload.getLinkedRecord("messageEdge")
-    const connection = ConnectionHandler.getConnection(store.get(conversation.__id), "Conversation_messages")
-    ConnectionHandler.insertEdgeAfter(connection, newMessageEdge)
+    const conversationStore = store.get(conversation.__id)
+
+    // TODO: Why is this not working?
```

Comment:
This is working now and should be removed.

Reply:
Oh yea!